Sources/SwiftProtobuf/Google_Protobuf_Duration_Extensions.swift

```diff
@@ -157,6 +157,11 @@ public extension Google_Protobuf_Duration {
 
 extension Google_Protobuf_Duration: ExpressibleByFloatLiteral {
     public typealias FloatLiteralType = Double
+    public init(seconds: Int64, nanos: Int32 = 0) {
```

Comment:
I'll add one for `Duration` for completeness. I added the one to `Timestamp` because it was used in a test and it seemed useful generally.